app/package.json

```diff
@@ -33,6 +33,7 @@
     "insight": "^0.8.3",
     "jquery": "^3.2.1",
     "lodash": "^4.17.4",
+    "mac-windows": "^0.5.4",
```

Comment:
You need to install it with Yarn, so it's added to the lockfile. Just run `yarn`.